__tests__/app.test.js

```diff
@@ -84,82 +84,165 @@ describe("GET", () => {
-          expect(comment).toHaveProperty('votes')
-          expect(comment).toHaveProperty('created_at')
-        })
-      })
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+          const { comments } = response.body;
+          comments.forEach((comment) => {
+            expect(comment).toHaveProperty("comment_id");
+            expect(comment).toHaveProperty("body");
+            expect(comment).toHaveProperty("article_id");
+            expect(comment).toHaveProperty("author");
+            expect(comment).toHaveProperty("votes");
+            expect(comment).toHaveProperty("created_at");
+          });
+        });
     });
     test("STATUS 200, should return an array of comments with the most recent comment first", () => {
       return request(app)
-      .get('/api/articles/1/comments')
-      .expect(200)
-      .then((response) => {
-        const { comments } = response.body;
-        expect(comments[0]['body']).toBe("This morning, I showered for nine minutes.")
-      })
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+          const { comments } = response.body;
+          expect(comments[0]["body"]).toBe(
+            "This morning, I showered for nine minutes."
+          );
+        });
     });
 
     test("STATUS 404 and should respond with a message when given an valid parametric endpoint but does not exist", () => {
       return request(app)
-      .get('/api/articles/999/comments')
-      .expect(404)
-      .then((response) => {
-        expect(response.body.msg).toBe('article does not exist')
-      })
-    })
+        .get("/api/articles/999/comments")
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toBe("article does not exist");
+        });
+    });
     test("STATUS 400 and should respond with a message when given an in-valid parametric endpoint", () => {
       return request(app)
-      .get('/api/articles/not-an-id/comments')
-      .expect(400)
-      .then((response) => {
-        expect(response.body.msg).toBe('Bad Request')
-      })
-    })
+        .get("/api/articles/not-an-id/comments")
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad Request");
+        });
+    });
   });
-  
 });
 
+//========================POST =========================
 
+describe("POST", () => {
+  describe("/api/articles/:article_id/comments", () => {
+    test("STATUS 201 and should insert a new comment with the article_id and return the comment back to the client", () => {
+      const newComment = {
+        username: "rogersop",
+        body: "Lobster pot",
+      };
+      return request(app)
+        .post("/api/articles/3/comments")
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+          const { Newcomment } = response.body;
+          const timeCreated_at = Newcomment[0]["created_at"];
```

Comment:
Please avoid using capitalised property keys

Reply:
I will change it now. Don't know how I didn't notice that.